### PR TITLE
Test video domain adapter

### PR DESCRIPTION
**Pull request by XianyuanLiu**
### Description
1. Create test_video_domain_adapter.
2. Remove some emails.
3. Change action_multi_domain to video_multi_domain.
4. Create test/helpers to save demo models & datasets (not sure if we need it).

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [x] New tests added to cover the changes.
- [ ] In-line docstrings updated.
- [x] [Source for documentation at `docs`](https://github.com/pykale/pykale/tree/main/docs/source) manually updated for new API.

### CODE REVIEW

File: tests/loaddata/test_digits_access.py [hunk 1]
```diff
@@ -19,6 +21,7 @@
 @pytest.mark.parametrize("source_name", SOURCES)
 @pytest.mark.parametrize("target_name", TARGETS)
 def test_get_source_target(source_name, target_name, download_path):
+    download_path = os.path.join(os.path.dirname(os.path.dirname(os.getcwd())), download_path)
```
[the remark this answers is not in the document]
Comment (XianyuanLiu) [reply tests/loaddata/test_digits_access.py:24]: I changed but some issues raised. I don't know if others use this, so I only change this file. I will try again to change the `conftest.py`

[the remark this answers is not in the document]
Comment (haipinglu) [reply tests/loaddata/test_digits_access.py:24]: Why do you make this change in the first place? It works well earlier and you did not change anything relevant to this module.

File: kale/pipeline/video_domain_adapter.py [hunk 2]
```diff
@@ -625,39 +659,24 @@ def critic_update_steps(self, batch):
                 h_s = torch.cat((h_s_rgb, h_s_flow), dim=1)
                 h_t = torch.cat((h_t_rgb, h_t_flow), dim=1)
 
+            # Need to improve to process rgb and flow dividedly in the future.
```
Comment (haipinglu) [inline kale/pipeline/video_domain_adapter.py:662]: individually or separately, not dividedly

File: kale/pipeline/video_domain_adapter.py [hunk 3]
```diff
@@ -625,39 +659,24 @@ def critic_update_steps(self, batch):
...
-                #     wasserstein_distance_rgb = (
-                #             critic_s_rgb.mean() - (1 + self._beta_ratio) * critic_t_rgb.mean()
-                #     )
-                #     wasserstein_distance_flow = (
-                #             critic_s_flow.mean() - (1 + self._beta_ratio) * critic_t_flow.mean()
-                #     )
-                #
-                #     critic_cost = (-wasserstein_distance_rgb + -wasserstein_distance_flow +
-                #                    self._gamma * gp_rgb + self._gamma * gp_flow) * 0.5
+                gp_rgb = losses.gradient_penalty(self.domain_classifier, h_s_rgb, h_t_rgb)
```
Comment (haipinglu) [inline kale/pipeline/video_domain_adapter.py:664]: what's `gp`? Gaussian process? `grad_pen(alty)`? Some abbreviations are hard to understand or remember.

**Review by haipinglu — APPROVED** [review]

**Pull request merged**